content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -336,6 +336,10 @@ The nodes are where your workloads (containers and Pods, etc) run. To add new no
   kubeadm join --token <token> <control-plane-host>:<control-plane-port> --discovery-token-ca-cert-hash sha256:<hash>
   ```
 
+{{< note >}}
+On Windows you should specify the `--cri-socket` parameter in the `kubeadm join` command. The default path is "npipe:////./pipe/containerd-containerd" for containerd.
```

Comment:
_default path_? This doesn't seem quite right.

Are we saying “if you chose containerd, then containerd is likely to make its socket available as `npipe:////./pipe/containerd-containerd`?

If so, that's not a default. It's not even a recommendation (we don't recommend a container runtime).

Reply:
That path is the value used by containerd if there is no gprc address specified to containerd (either in the config.toml or with `--address`/`-a` .

Reply:
OK. We should not document this as if it is a Kubernetes default.

Reply:
I think we should keep the note that on Windows (at least for now) the path needs to be specified but agree we can just say something like 'On Windows you should specify the `--cri-socket` parameter in the `kubeadm join` command to the gRPC path exposed by your container runtime.'

Reply:
Actually I don't think we need to keep the note.
https://github.com/kubernetes/kubernetes/pull/107317 updated the search paths used by kubelet to auto-detect the correct CRI socket ~5 months ago.
This should be sufficient.

Reply:
I've removed this note since it will be automatically discovered